main.go

```
@@ -0,0 +1,14 @@
+package main
+
+import (
+	"fmt"
+)
+
+func main() {
+	s := "gopher"
+	fmt.Println("Hello and welcome, %s!", s)
```

golangci-lint (1.64.8)

printf: fmt.Println call has possible Printf formatting directive %s

(govet)